google_authorized_keys/main_test.go

```diff
@@ -166,6 +196,11 @@ func TestGetUserKeysNew(t *testing.T) {
 	}
 
 	for count, tt := range tests {
+		replaceAttributesKeys(t, &tt.instanceMetadata, "[KEY]", randKey)
```

Comment:
Can you please take the opportunity that you are already touching theses tests and wrap these set tests in a t.Run()? You can use the slice index to name the test i.e. fmt.Sprintf("test-%d", índex)

Reply:
Done